src/main/java/org/fogbowcloud/manager/core/models/orders/NetworkOrder.java

```diff
@@ -1,55 +1,70 @@
 package org.fogbowcloud.manager.core.models.orders;
 
-import org.fogbowcloud.manager.core.models.orders.instances.OrderInstance;
+import org.fogbowcloud.manager.core.instanceprovider.InstanceProvider;
 import org.fogbowcloud.manager.core.models.token.Token;
 
 public class NetworkOrder extends Order {
 
-    private String gateway;
-    private String address;
-    private String allocation;
+	private String gateway;
+	private String address;
+	private String allocation;
 
-    public NetworkOrder(OrderState orderState, Token localToken, Token federationToken, String requestingMember,
-                        String providingMember, OrderInstance orderInstace, long fulfilledTime, String gateway,
-                        String address, String allocation) {
-        super(orderState, localToken, federationToken, requestingMember, providingMember, orderInstace, fulfilledTime);
-        this.gateway = gateway;
-        this.address = address;
-        this.allocation = allocation;
-    }
+	/**
+	 * Creating Order with predefined Id.
```

Comment:
If possible, remove the set methods. I guess you already analysed this problem (for another order class, right). If it is not possible to remove from here, please ack

Reply:
Done, [here](https://github.com/fogbow/fogbow-manager-core/pull/13/commits/b7c0996cc3f2df9c4767b870b63b3929b6ebce09).